### ISSUE TITLE
Add TP glossary terms as special weights for search engine

### ISSUE THREAD

**Issue by DinisCruz** [TeamMentor/TM_4_0_Design#667]
From spreadsheet in https://mail.google.com/mail/u/2/#inbox/14c7223e34bba8da we can use the TP (TeamProfessor) Glossary to improve the search results

Here is what it looks like (those summaries could also be really useful)

![image](https://cloud.githubusercontent.com/assets/656739/7012689/f583a372-dc80-11e4-98a1-9413873c5241.png)

Related to https://github.com/TeamMentor/TM_4_0_Design/issues/569

**Comment by roman87** [comment]
All of this should also be made into search items that we auto-fill in flare. But that is a different issue.

**Comment by DinisCruz** [comment]
yes, I was thinking of that too, but we will need a mapping of these terms to either:
* existing queries
* articles (i.e. these become queries)

**Comment by roman87** [comment]
So basically we can do a string search and add mapping this way. Not very accurate but I think until we have a tag system we can't really do much.
